### DISCUSSION TITLE
IMSAI8080 access point asking for password

### DISCUSSION THREAD

**Discussion by Apulo** [thehighnibble/thehighnibble.github.io#144] [Q&A]
Hello!

I am at the point where I can do the first test of the ESP32. I power it up and see an access point named IMSAI8080 however when I try to connect it asks for a password.

Is there something wrong or am I supposed to know a password? ;)

Cheers!

**Comment by Apulo** [comment]
You can disregard this, I connected via USB serial and everything passes.

**Comment by kernelpanic19** [comment]
If anyone else is looking, the initial password is documented here: https://thehighnibble.com/imsai8080/configure/#overview
imsai8080 SSID password: password
